src/main/java/com/labsynch/labseer/api/ApiPreferredIdController.java

```diff
@@ -32,7 +37,7 @@ public ResponseEntity<String> jsonArrayCheckCorpNameExists(@RequestBody String j
 		HttpHeaders headers = new HttpHeaders();
 		headers.add("Content-Type", "application/json");
 		Collection<PreferredNameDTO> preferredNameDTOs = PreferredNameDTO.fromJsonArrayToPreferredNameDTO(json);
-		preferredNameDTOs = PreferredNameDTO.getPreferredNames(preferredNameDTOs);
```

Comment:
I don't think this every really belonged on the DTO.  However, the real reason I moved it to lotService is because I needed access to the @Autowired properties utils service to be able to get access to the various acas configs during the requested name parsing.

Reply:
Yeah make sense. Thinking about it a bit more, I think I like option 2 more.  Mainly because it's less maintenance of data.  Manage all the updates deletes and additions to lot aliases when parent aliases change seems like a bit of a headache.

Reply:
Worked on this a bit but I realize now we have an issue when the lot corp name e.g. "LIT-00000001-0001" has no connection to the parent corp name which is the case on some system.  I think the best approach might be your second approach to finding the parent alias, combined with my initial abandoned approach (which lead to the refactor of the `generateParentLotName` in this PR.

-- Fetch all Parent Aliases in the DB, scan them against the incoming name for substring matches
-- From the matching parents, create a "Parent Alias Lot Corp Name" by passing in the alias as the "parent corp name" and the lot. which will generate what should be a matching string to the input string the user passed in.